### PR TITLE
Hide sensitive information in match output

### LINKED ISSUE #21351: Hide find-certificate output when using cert
### Discussed in https://github.com/fastlane/fastlane/discussions/21350

<div type='discussions-op-text'>

<sup>Originally posted by **RaedShari** June 21, 2023</sup>
### My current lane looks like:

  lane :beta do
     api_key = app_store_connect_api_key()
    increment_build_number()
    get_certificates(api_key: api_key)    
    get_provisioning_profile(api_key: api_key) 
    build_app(xcargs: "-allowProvisioningUpdates")
    upload_to_testflight(api_key: api_key)
  end

### When my GitLab CI/CD job runs, I can see it logs some sensitive contents "my certificates".

[08:52:33]: Creating authorization token for App Store Connect API
[08:52:33]: Fetching profiles...
[08:52:34]: Verifying certificates...
[08:52:34]: $ security find-certificate -a -c 'Apple Worldwide Developer Relations' -p /Users/ec2-user/Library/Keychains/login.keychain-db
[08:52:34]: ▸ -----BEGIN CERTIFICATE-----
[08:52:34]: ▸ ################################
[08:52:34]: ▸ ################################

### These logs are accessible by any user having access to the repo. Is there any option to silent this logs and preventing them from being logged?
</div>

### PR DESCRIPTION
**Pull request by oct0f1sh**
### Checklist

- [x] I've run `bundle exec rspec` from the root directory to see all new and existing tests pass
- [x] I've followed the _fastlane_ code style and run `bundle exec rubocop -a` to ensure the code style is valid
- [x] I see several green `ci/circleci` builds in the "All checks have passed" section of my PR ([connect CircleCI to GitHub](https://support.circleci.com/hc/en-us/articles/360008097173-Why-aren-t-pull-requests-triggering-jobs-on-my-organization-) if not)
- [x] I've read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)
- [x] I've updated the documentation if necessary.
- [x] I've added or updated relevant unit tests.

### Motivation and Context

I've noticed that when running _match_, it tends to output the full WWDR certificate to the console. This has the potential to leak sensitive information, and I would prefer this not be directly viewable in CI logs. This was discussed in issues #21350 and #21351 (duplicate issues). 

### Description

As mentioned in a comment by user @AnubisFup, you can silence this output by passing in `print: false` to the backticks action. I added their fix, along with updating tests to expect the `print` parameter. I tested using `bundle exec rspec` as well as `bundle exec fastlane test`

### Testing Steps

In a project that uses _match_ for managing certificates, download your certificates as you usually would. Before this change, you would see your certificates being output to the console in bright purple text. Now, that output should be hidden.

### CODE REVIEW

File: fastlane/swift/FastlaneSwiftRunner/FastlaneSwiftRunner.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -12,6 +12,7 @@
 		1257253924B7992C00E04FA3 /* main.swift in Sources */ = {isa = PBXBuildFile; fileRef = 1257253824B7992B00E04FA3 /* main.swift */; };
 		1267C3F42773A43E004DE48A /* Atomic.swift in Sources */ = {isa = PBXBuildFile; fileRef = 1267C3F32773A43E004DE48A /* Atomic.swift */; };
 		12D2EB8D2620D83C00844013 /* OptionalConfigValue.swift in Sources */ = {isa = PBXBuildFile; fileRef = 12D2EB8C2620D83B00844013 /* OptionalConfigValue.swift */; };
+		5F5D0ED98E02119343C05257 /* FastlaneRunner in FastlaneRunnerCopySigned */ = {isa = PBXBuildFile; fileRef = D556D6A91F6A08F5003108E3 /* FastlaneRunner */; settings = {ATTRIBUTES = (CodeSignOnCopy, ); }; };
```
Comment (oct0f1sh) [inline fastlane/swift/FastlaneSwiftRunner/FastlaneSwiftRunner.xcodeproj/project.pbxproj:15]: I think this file was changed when I ran unit tests. Should I revert the changes to this file?

**Review by getaaron — COMMENTED** [review]
Looks great 👍 

Could you please revert the `project.pbxproj` changes? If that needs to be updated we should do it in a separate PR

Also if you can, please follow the instructions to oauth CircleCI to GitHub so the automated tests run

**Comment by oct0f1sh** [comment]
> Looks great 👍
> 
> Could you please revert the `project.pbxproj` changes? If that needs to be updated we should do it in a separate PR
> 
> Also if you can, please follow the instructions to oauth CircleCI to GitHub so the automated tests run

Just reverted the file in d995a7a, and hopefully my CircleCI link should be working 🤞 

Thank you! 😁

File: fastlane/swift/FastlaneSwiftRunner/FastlaneSwiftRunner.xcodeproj/project.pbxproj (hunk 1 above)
Comment (oct0f1sh) [reply fastlane/swift/FastlaneSwiftRunner/FastlaneSwiftRunner.xcodeproj/project.pbxproj]: Removed in d995a7a

**Review by getaaron — APPROVED** [review]
